### PR TITLE
OSOE-70: Adapting to Helpful Libraries refactoring

### PR DESCRIPTION
**Pull request by Piedone**
OSOE-70

### CODE REVIEW

File: Activities/ValidatePrivacyConsentCheckboxTask.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public class ValidatePrivacyConsentCheckboxTask : TaskActivity
         if (await _consentService.IsUserAcceptedConsentAsync(_hca.HttpContext))
             return Outcomes("Done", "Valid");
 
-        var consentCheckboxName = $"{nameof(PrivacyConsentCheckboxPart)}.{nameof(PrivacyConsentCheckboxPart.ConsentCheckbox)}";
+        const string consentCheckboxName = $"{nameof(PrivacyConsentCheckboxPart)}.{nameof(PrivacyConsentCheckboxPart.ConsentCheckbox)}";
```
Comment (sarahelsaig) [inline Activities/ValidatePrivacyConsentCheckboxTask.cs:56]: [OMG!!](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-10#constant-interpolated-strings)
image

Comment (Piedone) [reply Activities/ValidatePrivacyConsentCheckboxTask.cs:56]: No, O**M**G!

**Review by sarahelsaig — APPROVED** [review]

File: Activities/ValidatePrivacyConsentCheckboxTask.cs (hunk 1 above)
Comment (I3undy) [reply Activities/ValidatePrivacyConsentCheckboxTask.cs:56]: OMG!

Comment (Piedone) [reply Activities/ValidatePrivacyConsentCheckboxTask.cs:56]: OOG?

**Pull request merged**